### PR TITLE
Feature/9316 scanner tab/9632 popover

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
Adds a tooltip on first appearance that explains the newly introduced universal QR code scanner button in the tab bar.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9632

## Screenshots
Simulator Screen Shot - iPod touch (7th generation) - 2021-09-30 at 18 09 23
Simulator Screen Shot - iPod touch (7th generation) - 2021-09-30 at 18 09 33

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/QRScanner/Tooltip/QRScannerTooltipViewController.xib [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/QRScanner/Tooltip/QRScannerTooltipViewController.xib:1]: I fully understand this code 😏💅

File: src/xcode/ENA/ENA/Source/Scenes/QRScanner/Tooltip/QRScannerTooltipViewController.xib [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                        <constraint firstAttribute="width" relation="lessThanOrEqual" constant="200" id="asg-CZ-u6Y"/>
+                    </constraints>
+                    <fontDescription key="fontDescription" style="UICTFontTextStyleHeadline"/>
+                    <color key="textColor" name="ENA Text Primary 1 Color"/>
+                    <nil key="highlightedColor"/>
+                    <userDefinedRuntimeAttributes>
+                        <userDefinedRuntimeAttribute type="string" keyPath="ibEnaStyle" value="headline"/>
+                    </userDefinedRuntimeAttributes>
+                </label>
+                <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="CUs-sA-78T">
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/QRScanner/Tooltip/QRScannerTooltipViewController.xib:37]: Looks like the scroll view is partly overlapping the bounds of the button. I don't know, maybe its still enough space to tap the button 🤷‍♂️

Bildschirmfoto 2021-10-01 um 10 12 30

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/QRScanner/Tooltip/QRScannerTooltipViewController.xib (hunk 2 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/QRScanner/Tooltip/QRScannerTooltipViewController.xib:37]: I put the button on top so it always stays tappable

Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/QRScanner/Tooltip/QRScannerTooltipViewController.xib:37]: Good catch Artur :D

File: src/xcode/ENA/ENA/Source/Workers/Store/Store.swift [hunk 3]
```diff
@@ -45,6 +45,10 @@ protocol StoreProtocol: AnyObject {
 	/// We only inform the user once. By default the value of this property is `true`.
 	var userNeedsToBeInformedAboutHowRiskDetectionWorks: Bool { get set }
 
+	/// `true` if the user needs to be shown the QR code scanner tooltip.
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Workers/Store/Store.swift:48]: 👍

**Review by naveeddotio — APPROVED** [review]

**Review by CV113 — APPROVED** [review]
UA approved

**Pull request merged**